include/dav1d/picture.rs

```diff
@@ -82,7 +82,7 @@ pub struct Dav1dPicture {
     pub mastering_display: Option<NonNull<Rav1dMasteringDisplay>>,
     pub itut_t35: Option<NonNull<Dav1dITUTT35>>,
     pub n_itut_t35: usize,
-    pub reserved: [uintptr_t; 3],
+    pub reserved: [uintptr_t; 4],
```

Comment:
I'm curious where this extra room came from.

Reply:
Don't know. This was reduced to 3 when `n_itut_t35` was added and now it's changed back to 4.

Reply:
Yeah I saw it was reduced then which made sense because the new field used up one of them, but the change here doesn't change the size.